test/src/pick_datetime_test.dart

```diff
@@ -182,6 +182,24 @@ void main() {
         });
       });
 
+      test('parse DateTime with timezone +0230', () {
+        const input = '2023-01-09T12:31:54+0230';
```

Comment:
Does `DateTime.tryParse` handle trailing spaces?

Reply:
Yes, added a test for it